docs/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: default
+title: Home
+---
+
+This pipeline can be used to process RNA sequenced by either the Pacific Biosciences or Oxford Nanopore sequencer, starting from FastQ files. It will perform mapping to a reference genome (using minimap2), INDEL/mismatch and noncanonical splice junction correction (using TranscriptClean) and identify and count known and novel genes/transcripts (using TALON).
+
+This pipeline is part of [BioWDL](https://biowdl.github.io/)
+developed by the SASC team at [Leiden University Medical Center](https://www.lumc.nl/).
+
+## Usage
+You can run the pipeline using
+[Cromwell](http://cromwell.readthedocs.io/en/stable/):
+
+```bash
+java -jar cromwell-<version>.jar run -i inputs.json pipeline.wdl
+```
+
+### Inputs
+Inputs are provided through a JSON file. The minimally required inputs are
+described below, but additional inputs are available.
+A template containing all possible inputs can be generated using
+Womtool as described in the
+[WOMtool documentation](http://cromwell.readthedocs.io/en/stable/WOMtool/).
+
+```json
+{
+    "Pipeline.sampleConfigFile": "A sample configuration file (see below).",
+    "Pipeline.outputDirectory": "The path to the output directory.",
+    "Pipeline.annotationGTF": "GTF annotation containing genes, transcripts, and edges.",
```

Comment:
edges or exons?

Reply:
I took this line from the TALON documentation.
Edges can be either exons or introns in the TALON database, these are stored in the "edge" table.